dfm/modelling/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from dfm.modelling.utils.utils import *  # noqa
```

Comment:
We shouldn't use import * especially not in __init__ files because it makes tracing code source problematic.

A folder for 1 script? I would remove it.

Reply:
Perhaps we should just include a general utils folder?

Reply:
As long as it contains more than one script I am down with that, but I actually think it is better to keep the util script local (i.e. utils within their respective modules). However, I don't have a strong commitment either way.

Reply:
Okay fair enough.